### PR TITLE
My Updates during class

### PR DESCRIPTION
**Pull request by BenFranco47**
My work

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,7 +5,19 @@ const pc = newPlayableCharacter(100, 110)
 const npc = newNonPlayableCharacter(50, 300)
 
 // have the NPC start walking east immediately
-npc.walkEast()
+npc.walkNorth(1400, () => {
```
Comment (Dvogel345) [inline index.js]: This looks great

File: newNonPlayableCharacter.js [hunk 2]
```diff
@@ -56,4 +56,15 @@ function newNonPlayableCharacter(x, y) {
         walkSouth: walkSouth,
         stop: stop
     }
+}
+
```
Comment (Dvogel345) [inline newNonPlayableCharacter.js:112]: Suggestion: making a note about what this function is would help readers see what happening regardless of how easy it is to read

File: newNonPlayableCharacter.js [hunk 3]
```diff
@@ -23,6 +23,10 @@ function newNonPlayableCharacter(x, y) {
 
     setInterval(moveCharacter, 1)
 
+
+
+
+
```
Comment (Dvogel345) [inline newNonPlayableCharacter.js:29]: Adding a note similar to the rest of the functions about what direction this code does would help

File: index.js [hunk 4]
```diff
@@ -5,20 +5,27 @@ const pc = newPlayableCharacter(100, 110)
 const npc = newNonPlayableCharacter(50, 300)
 
 // have the NPC start walking east immediately
-npc.walkNorth(1400, () => {
-    npc.walkEast(1200, () => {
+
```
Comment (Dvogel345) [inline index.js:8]: Great job, the code looks great compared to how the first commit looks like

Comment (Dvogel345) [reply index.js:8]: Making a better commit name, such as "Updated North, South, East, and West NPC function direction paths" would help reviewer know what was committed before-hand

**Review by Dvogel345 — COMMENTED** [review]
review my review please

**Review by Dvogel345 — COMMENTED** [review]
Review this review thankyou, please

**Comment by hinchley2018** [comment]
This was meant to be created off of [my fork](https://github.com/hinchley2018/JS-Asynchrony-With-Callback-Functions) closing so it doesn't affect curriculum

**Pull request closed without merging**